### ISSUE TITLE
Could `cols_hide()` hide a new nanoplots column?

### ISSUE THREAD

**Issue by olivroy** [rstudio/gt#1574]
## Prework

* [x] Read and agree to the [code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html) and [contributing guidelines](https://github.com/rstudio/gt/blob/master/.github/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/rstudio/gt/issues), whether open or closed, comment on the existing thread instead of posting a new issue.

## Description

nanoplots work well with html, but not so much with docx.

![image](https://github.com/rstudio/gt/assets/52606734/353d8648-bb38-4b68-adb5-b554d39ea442)

```r
library(gt)
my_gt <- illness |>
  dplyr::slice_head(n = 10) |>
  gt::gt(rowname_col = "test") |>
  cols_nanoplot(
    columns = starts_with("day"),
    new_col_name = "nanoplots",
    new_col_label = md("*Progression*")
  )
gtsave(my_gt, "xy.html")
gtsave(my_gt |> cols_hide("nanoplots"), "xy.docx")
#> Error in `build_row_styles()`:
#> ! `build_row_styles()` was called with invalid `colnum` values.
```

## Expected result

Either nanoplots should be hidden in docx output, or `cols_hide()` work with nanoplots?

## Session info

latest gt 0.10.1

**Comment by olivroy** [comment]
Relates to #1550

**Comment by rich-iannone** [comment]
I think there ought to be a static graphic that can be rendered for the non-HTML formats. This will probably involve using rsvg to generate an image suitable for the different output formats. In addition, there should be some options in `nanoplot_options()` for generating a static nanoplot (and setting what gets shown).

What do you think? Perhaps we could skip to that work and avoid the temporary fix of automatically hiding the column. I'll create another issue detailing the work to be done if it sounds like a good plan.

**Comment by olivroy** [comment]
Agreed it would be great to have nanoplots image. But a first step would be to hide them by default for now outside html context. (That will require less work?)

Afterwards, create the code to transform them to images that can be embedded?

**Comment by rich-iannone** [comment]
I think I'm seeing the light with your plan! Upon reconsidering, the auto-hiding of nanoplots would be a simple PR, and the subsequent work to make plots static image will likely take considerable time (i.e., I might not even get to it until late August).
